CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.7.0 - 2021-05-12
+### Added
+- Project validation prior to `Create Static Web App...`
+- After creating a Static Web App, `git pull` is executed to retrieve workflow config
```

Comment:
> workflow config

I don't know why I waited until now to look up the "correct" term for the workflow file... but it looks to be "GitHub Actions workflow": https://docs.microsoft.com/en-us/azure/static-web-apps/github-actions-workflow

Reply:
```suggestion
- After creating a Static Web App, `git pull` is executed to retrieve the [GitHub Actions workflow file](https://docs.microsoft.com/en-us/azure/static-web-apps/github-actions-workflow)
```